### DISCUSSION TITLE
In Taiko, a longer beatmap shouldn't mean a higher HP drain

### DISCUSSION THREAD

**Discussion by jlefebvre1997** [ppy/osu#24002] [Ideas]
Title. It doesn't make any sense at all

**Comment by peppy** [comment]
Are you talking about lazer or stable or both?

**Comment by 120-cell** [comment]
Seconded. This is definitely a problem in lazer.
For example, here are a 93.9% acc fail on a long map and a 73.7% pass on a short map. Both maps are HP4.5
![image](https://github.com/user-attachments/assets/880de9c2-7dd9-407c-9fe8-821565b6801d)
![image](https://github.com/user-attachments/assets/43defbf7-43e4-46d5-867d-edc6ad622eed)


I don't have experience with it on stable, but I've seen others mention it, likely referring to stable https://osu.ppy.sh/community/forums/topics/1752253?n=1.

**Reply by peppy** [reply]
Are you able to attach the two replays for reference? Also from your reply it sounds like you're confirming that this is also a concern in stable?

**Reply by 120-cell** [reply]
here are the two replays (and the HP4.5 Edit of Nyan Cat, in case it's required to view the replay)
[replays.zip](https://github.com/user-attachments/files/19591822/replays.zip)

I also confirmed that the same issue appears on stable:
![fail](https://github.com/user-attachments/assets/936c34a3-0554-4469-98b4-ed7dbc614011)
![pass](https://github.com/user-attachments/assets/e1250f2c-676a-402f-9245-7c6880ecb7fa)
here are the corresponding replays (and the map edit):
[stable replays.zip](https://github.com/user-attachments/files/19591877/stable.replays.zip)
